guides/plugins/plugins/framework/message-queue/add-message-handler.md

```diff
@@ -49,7 +53,14 @@ There is a console command to start a worker that will receive incoming messages
 bin/console messenger:consume async
 ```
 
-Where `async` is the transport you want to consume message from. There is also an API-Route that lets you consume messages for a given transport. Just post to the route `/api/_action/message-queue/consume` and define the transport from which you want to consume:
+You can also append more transports to the command, so the worker will consume messages from multiple transports which will result in prioritisation as mentioned in [Prioritized Transports](https://symfony.com/doc/current/messenger.html#prioritized-transports):
+
+```bash
+// 
+bin/console messenger:consume async async_low_priority
+```
+
+Where `async` and `async_low_priority` are the transports you want to consume messages from. There is also an API-Route that lets you consume messages for a given transport. Just post to the route `/api/_action/message-queue/consume` and define the transport from which you want to consume:
```

Comment:
I think the API route is meant to be used only by the admin worker? 🤔 

I would leave this sentence out.

Reply:
Yes but in the command / sentence above I describe how to consume messages from multiple transports and the route only takes one transport instead of multiple ones  :sweat_smile: